RosBE-Unix/Base-i386/RosBE-Builder.sh

```diff
@@ -58,19 +58,32 @@ echo "ReactOS."
 echo
 
 if [ "$1" = "-h" ] || [ "$1" = "-?" ] || [ "$1" = "--help" ]; then
-	echo "Syntax: ./RosBE-Builder.sh [installdir]"
+	echo "Syntax: ./RosBE-Builder.sh [prefixdir] [destdir]"
 	echo
-	echo " installdir - Optional parameter to specify an installation directory. If you"
-	echo "              do this, the script will check whether this directory does not"
-	echo "              yet exist and in this case, it will perform an unattended"
-	echo "              installation to this directory."
+	echo " prefixdir  - Optional parameter to specify an prefix directory. This parameter"
+	echo "              will be passed to the  tools' configuration  as  the  argument of"
+	echo "              '--prefix'."
+	echo "              [default: '$DEFAULT_INSTALL_DIR']"
+	echo " destdir    - Optional parameter to specify an destination directory as  system"
+	echo "              root path to install ReactOS build environment. This parameter is"
+	echo "              only used when you want to build RosBE but  not  install it right" 
+	echo "              away, e.g. for build a sofware package. For who want  to  install"
+	echo "              RosBE directly, just leave it as default."
+	echo "              [default: '/']"
 	echo
-	echo "Usually, you just call the script without any parameters and it will guide you"
+	echo "This script will check whether the '\$destdir\$prefixdir' does  not  yet  exist"
+	echo "and in this case, it will treat this directory as root  path  then  perform  an"
+	echo "unattended installation to this directory."
+	echo
+	echo "Usually, you just call the script without any parameters and it will guide  you"
```

Comment:
Question: Are the extraneous spaces sprinkled through the sentences made in order to give the appearance of the displayed text to be "justified" ?

Reply:
Yes. Shoud it use another layout scheme?